lib/config/rules/rule-helpers/console-methods.js

```diff
@@ -7,7 +7,6 @@ module.exports = [
   'debug',
   'dir',
   'dirxml',
-  'error',
```

Comment:
By this logic we should also remove `info` and `warn`, since we have code that needs to showcase those, too. I would prefer just no `console` methods, period, or disable the linter to get them. Console just gives people the feeling they have addressed the possible error case when they haven't at all.